platform/local/dnsmasq.go

```diff
@@ -108,6 +111,63 @@ func newInterface(s byte, i uint16) *Interface {
 	}
 }
 
+// configureNAT creates and append a NAT rule
+// using the interface i as output
+func configureNAT(i string) error {
+	table, err := iptables.New()
+	if err != nil {
+		return fmt.Errorf("unable to get iptables: %w", err)
+	}
+
+	if err := table.AppendUnique("nat", "POSTROUTING", "-j", "MASQUERADE", "-o", i); err != nil {
+		return fmt.Errorf("unable to append rule: %w", err)
+	}
+
+	return nil
+}
+
+// generateVethPair creates and returns a map holding
+// a pair of veth based on the pid
+// where:
+// [0] is the network namespace veth
+// [1] is the host namespace veth
+func generateVethPair(pid int) []string {
+	x := pid%(31-16) + 16
```

Comment:
```suggestion
	x := (pid/256)%(31-16) + 16
```
here the division approach